nav2_smac_planner/include/nav2_smac_planner/a_star.hpp

```diff
@@ -69,6 +71,18 @@ class AStarAlgorithm
 
   typedef std::priority_queue<NodeElement, std::vector<NodeElement>, NodeComparator> NodeQueue;
 
+  /**
+   * @struct nav2_smac_planner::GoalState
+   * @brief A struct to store the goal state
+   */
+  struct GoalState
```

Comment:
Can this go into types.hpp along wit hthe typedef vector?

Reply:
Still open :smile:

Reply:
done